### ISSUE TITLE
PhpStorm / WebStorm 2016.3

### ISSUE THREAD

**Issue by bchatard** [bchatard/jetbrains-alfred-workflow#9]
Q | A
--- | ---
Workflow version | _1.0.9_
OSX Version | _all_
Alfred Version | _2 / 3_
JetBrains Product/Version | _PhpStorm / 2016.3_ \| _WebStorm / 2016.3_
Installation type | _classic .dmg & Caskroom_

### Description
can't open more than once projet

### Why?
1. PhpStorm change bin name `pstorm` ==> `phpstorm`
2. WebStorm change bin name `wstorm` ==> `webstorm`

### Solution
1. PhpStorm
Generate bin with old name (`pstorm`) or update workflow to use `phpstorm` instead of `pstorm`

2. WebStorm
Generate bin with old name (`wstorm`) or update workflow to use `webstorm` instead of `wstorm`

Update workflow (see [customisation](https://github.com/bchatard/jetbrains-alfred-workflow#customisation))

**Comment by felipep** [comment]
thanks for the solution, I did something similar:
* I created a new command line launcher (tools=> Create new command line launcher) path '/usr/local/bin/phpstorm'
* and then changed the script inside the pstorm workflow to:

````
PATH=$PATH:/usr/local/bin
. ./main.zsh `which phpstorm` {query}
````

**Comment by bchatard** [comment]
@felipep thanks for the feedback

**Comment by bchatard** [comment]
closed by #11
